### PR TITLE
LablTk 8.06.14 and OCamlBrowser 5.2.0

### PR DESCRIPTION
**Pull request by garrigue**
New release for compatibility with OCaml 5.2 and C2x.
Old versions appear to be broken.

### CODE REVIEW

File: packages/labltk/labltk.8.06.14/opam [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+install: [
+  [make "install"]
+]
+depends: [
+  "ocaml" {>= "4.08"}
+  "ocamlfind" {build}
+  "conf-tcl"
+  "conf-tk"
+]
+post-messages: [
```
Comment (mseri) [inline packages/labltk/labltk.8.06.14/opam:21]:
```suggestion
conflicts: [ "ocaml-option-bytecode-only" ]
post-messages: [
```

Comment (mseri) [reply packages/labltk/labltk.8.06.14/opam:21]: For
```
# cd support; make opt
# make[1]: Entering directory '/home/opam/.opam/5.2/.opam-switch/build/labltk.8.06.14/support'
# /home/opam/.opam/5.2/bin/ocamlopt  -c -I +unix support.ml
# make[1]: /home/opam/.opam/5.2/bin/ocamlopt: No such file or directory
# make[1]: *** [Makefile:86: support.cmx] Error 127
# make[1]: Leaving directory '/home/opam/.opam/5.2/.opam-switch/build/labltk.8.06.14/support'
# make: *** [Makefile:43: libraryopt] Error 2

File: packages/labltk/labltk.8.06.14/opam [hunk 2]
```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+maintainer: "garrigue@math.nagoya-u.ac.jp"
+authors: ["Jacques Garrigue et al., Nagoya University"]
+homepage: "https://garrigue.github.io/labltk/"
+bug-reports: "https://github.com/garrigue/labltk/issues"
+dev-repo: "git+https://github.com/garrigue/labltk.git"
+license: "LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception"
+build: [
+  ["./configure" "-use-findlib" "-verbose" "-installbindir" bin]
+  [make "library" "opt"]
```
Comment (mseri) [inline packages/labltk/labltk.8.06.14/opam:10]:
```suggestion
  [
    make
    "library"
    "opt" {!ocaml-option-bytecode-only:installed}
  ]
```

Comment (mseri) [reply packages/labltk/labltk.8.06.14/opam:10]: Otherwise we could try this

**Comment by garrigue** [comment]
@mseri Thanks, I didn't know of this fine grain way to pass command line arguments to make.

**Comment by garrigue** [comment]
Why is it that the tests do not run after merging your suggested change ?

**Comment by mseri** [comment]
Seems like the CI is broken

**Comment by mseri** [comment]
Thanks!

**Pull request merged**